docker/bootstrap/Dockerfile.mysql57-arm64v8

```diff
@@ -0,0 +1,40 @@
+FROM vitess/bootstrap:common
+
+# Install MySQL 5.7
+RUN add-apt-repository 'deb http://ftp.debian.org/debian sid main' && \
+apt-get update && DEBIAN_FRONTEND=noninteractive apt-get install -y --no-install-recommends libmysqlclient-dev mysql-client-5.7 mysql-server-5.7 \
+cmake \
+apt-utils \
+libaio-dev \
+libncurses5-dev \
+libssl-dev \
+zlib1g-dev \
+libgcrypt11-dev \
+libev-dev \
+libcurl4-gnutls-dev \
+vim-common \
+bison \
+flex \
+wget \
+libdbd-mysql-perl \
+rsync \
+libev4 \
+python3-distutils-extra  && \
+rm -rf /var/lib/apt/lists/* && \
+wget https://github.com/percona/percona-xtrabackup/archive/percona-xtrabackup-2.4.13.tar.gz -P /opt && \
+tar zxf /opt/percona-xtrabackup-2.4.13.tar.gz -C /opt && \
+rm /opt/percona-xtrabackup-2.4.13.tar.gz && \
+cd /opt/percona-xtrabackup-percona-xtrabackup-2.4.13 && \
+mkdir bld && \
+cd bld && \
+cmake .. -DBUILD_CONFIG=xtrabackup_release -DWITH_MAN_PAGES=OFF -DDOWNLOAD_BOOST=1 -DWITH_BOOST=/usr/local && \
+make -j4 && \
+make install
```

Comment:
This and all the build dependencies are going to be substaintial. It would be better to build this in a different container then then use `COPY --from=...` to just copy the needed binaries and dependencies. That will also prevent you have having to modify the path below

Reply:
@dkhenry @morgo Sorry for my response too late.  I have rewritten the docker file,  and use multi-stages to build Xtrabackup. Thanks for you tell me this method. The built images is smaller a lot than before.   But the error still exist.  Is this because the system unstable?
```
github.com/hashicorp/serf@v0.0.0-20161207011743-d3a67ab21bc8: stream error: stream ID 913; INTERNAL_ERROR
github.com/klauspost/compress@v0.0.0-20180801095237-b50017755d44: stream error: stream ID 1003; INTERNAL_ERROR
golang.org/x/image@v0.0.0-20190227222117-0694c2d4d067: stream error: stream ID 929; INTERNAL_ERROR
##[error]Process completed with exit code 1.
```